pytest_selenium/drivers/browserstack.py

```diff
@@ -2,6 +2,12 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+try:
+    import configparser
+except ImportError:
```

Comment:
Hey @davehunt a small comment here: in my experience it is better to try to import using a version check rather than fallback from an import error, because the latter might make it harder to debug some import errors. For example, if one has a botched installation or someone has mocked `configparser` somehow and it fails to import, it will fallback to the second import and "hide" the original error (in Python 2 at least, in Python 3 it won't be a big deal). I had this issue in pytest-dev/pytest-mock#68, where `import mock` was failing because some missing dependencies.

In summary, I suggest using this instead:

```python
if sys.version_info[0] == 2:
    import ConfigParser as configparser
else:
    import configparser
```